### PR TITLE
Curoo changes

### PR DESCRIPTION
**Pull request by benjaminboxler**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '8.0'
+# Uncomment this line if you're using Swift
+use_frameworks!
+
+
+target 'OAuthSwiftTests' do
+
+  pod 'OHHTTPStubs'
```
Comment (sdduursma) [inline Podfile]: Can you fix the version for OHHTTPStubs ?

File: OAuthSwiftTests/OAuthSwiftTests.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//  OAuthSwiftTests.swift
+//  OAuthSwiftTests
+//
+//  Created by Benjamin Boxler on 26/10/2015.
+//  Copyright © 2015 Dongri Jin. All rights reserved.
+//
+
+import XCTest
+
+class OAuthSwiftTests: XCTestCase {
```
Comment (sdduursma) [inline OAuthSwiftTests/OAuthSwiftTests.swift]: This doesn't do anything currently, can we remove it?

File: OAuthSwiftTests/OAuthSwiftHTTPRequestTests.swift [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+//  Copyright © 2015 Dongri Jin. All rights reserved.
+//
+
+import XCTest
+@testable import OAuthSwift
+import OHHTTPStubs
+
+class OAuthSwiftHTTPRequestTests: OAuthSwiftTests {
+    
+    override func setUp() {
```
Comment (sdduursma) [inline OAuthSwiftTests/OAuthSwiftHTTPRequestTests.swift]: Remove

File: OAuthSwiftTests/OAuthSwiftHTTPRequestTests.swift [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    
+    override func tearDown() {
+        OHHTTPStubs.removeAllStubs()
+        super.tearDown()
+    }
+    
+    func testFailure() {
+        let oAuthSwiftHTTPRequest = OAuthSwiftHTTPRequest(URL: NSURL(string: "http://127.0.0.1")!)
+        let failureExpectation = expectationWithDescription("Expexted `failure` to be called")
+        oAuthSwiftHTTPRequest.start()
```
Comment (sdduursma) [inline OAuthSwiftTests/OAuthSwiftHTTPRequestTests.swift]: Can you call `start()` after setting the handlers ?

File: OAuthSwiftTests/OAuthSwiftHTTPRequestTests.swift [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+            }, withStubResponse: { _ in
+                return OHHTTPStubsResponse(
+                    data: NSString(string: "Success!").dataUsingEncoding(NSUTF8StringEncoding)!,
+                    statusCode: 200, headers: nil)
+
+        })
+
+        let oAuthSwiftHTTPRequest = OAuthSwiftHTTPRequest(URL: NSURL(string: "http://127.0.0.1")!)
+        let successExpectation = expectationWithDescription("Expexted `failure` to be called")
+        oAuthSwiftHTTPRequest.start()
```
Comment (sdduursma) [inline OAuthSwiftTests/OAuthSwiftHTTPRequestTests.swift]: idem

File: OAuthSwiftTests/OAuthSwiftTests.swift (hunk 2 above)
Comment (benjaminboxler) [inline OAuthSwiftTests/OAuthSwiftTests.swift]: Gonna keep it so we have a base class for any global setup

**Pull request merged**